hc-venv/.Python

```diff
@@ -0,0 +1 @@
+/usr/local/Cellar/python3/3.6.3/Frameworks/Python.framework/Versions/3.6/Python
```

Comment:
This file should not be in the repo

Reply:
Issue has been resolved, there was an issue with the naming of my env in the gitignore file